packages/bot-web-ui/src/components/bot-notification/bot-notification-utils.ts

```diff
@@ -20,13 +20,14 @@ export type TAction = {
 };
 
 export type TNotificationStyle = {
```

Comment:
```suggestion
export type TNotificationStyle = {
```
First question - why is all properties here optional? If you want it to be optional for certain cases, u can just use `Partial<TNotificationStyle>`

Reply:
Sure, let me check the usage.

Reply:
Issue resolved ✋